lib/ExpensiMark.d.ts

```diff
@@ -1,11 +1,13 @@
-declare type Replacement = (...args: string[]) => string;
+declare type Replacement = (...args: string[], extras?: ExtrasObject) => string;
 declare type Name =
     | 'codeFence'
     | 'inlineCodeBlock'
     | 'email'
     | 'link'
     | 'hereMentions'
+    | 'roomMentions'
```

Comment:
I'm confused, why we need the `roomMentions` here if we renamed it o `reportMentions`?

Reply:
Thanks! Missed that while implementing